ts/features/bonus/cgn/store/reducers/categories.ts

```diff
@@ -3,14 +3,24 @@ import { createSelector } from "reselect";
 import { Action } from "../../../../../store/actions/types";
 import { GlobalState } from "../../../../../store/reducers/types";
 import { ProductCategoryEnum } from "../../../../../../definitions/cgn/merchants/ProductCategory";
-import { cgnSelectedCategory } from "../actions/categories";
+import { cgnCategories, cgnSelectedCategory } from "../actions/categories";
+import {
+  remoteError,
+  remoteLoading,
+  remoteReady,
+  remoteUndefined,
+  RemoteValue
+} from "../../../bpd/model/RemoteValue";
+import { NetworkError } from "../../../../../utils/errors";
 
 export type CgnCategoriesState = {
   selectedCategory: ProductCategoryEnum | undefined;
+  list: RemoteValue<ReadonlyArray<ProductCategoryEnum>, NetworkError>;
```

Comment:
Since in this case it is not an one shot request but the category list may already exist, could be worth, maybe in another iteration, use `pot` instead of `RemoteValue`? Using the pot the user would avoid seeing all the elements disappear from the list during the pull to refresh.

https://user-images.githubusercontent.com/26501317/153917620-5a005962-8cae-4fcb-be33-fa973de77230.mp4

Reply:
nice point, thanks for noticing I think a pot may be really useful in this case...i'm changing it